src/shared/components/comment/comment-node.tsx

```diff
@@ -296,6 +296,12 @@ export class CommentNode extends Component<CommentNodeProps, CommentNodeState> {
             className={classNames({
               "ms-2": !this.props.noIndent,
             })}
+            onClick={linkEvent(this, this.handleCommentCollapse)}
+            onKeyDown={linkEvent(this, this.handleCommentCollapse)}
```

Comment:
This will collapse on any key down?

Reply:
Hmm you're right! I added it because I was getting:

error  Visible, non-interactive elements with click handlers must have at least one keyboard listener  jsx-a11y/click-events-have-key-events

I removed the onKeyDown and turned the div into a button:
https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/click-events-have-key-events.md

Reply:
If you want to have it work with a keyboard shortcut still, you could always create a function that checks which keys are being pressed (something like ctrl or meta + something else; consult the `MarkdownTextArea` component for shortcuts that are already being used so as to avoid collision).

Reply:
Ok turning it into a button made it worse.. 

would this work for accessibility ?
<div onClick{() => {}} aria-hidden="true" />